### PR TITLE
Add training loss plot scripr

### PR DESCRIPTION
**Pull request by datnt88**
- code for generate  training loss

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -33,6 +46,10 @@ ENV FVCORE_CACHE="/tmp"
 
 WORKDIR /detectron2_repo
 
+#RUN gsutil cp gs://hptuning/split_damages.zip .
+#RUN unzip split_damages.zip
```
Comment (datnt88) [inline Dockerfile:50]: you should specific dataset rather than "split_damages"

File: Dockerfile [hunk 2]
```diff
@@ -33,6 +46,10 @@ ENV FVCORE_CACHE="/tmp"
...
 #COPY split_damages .
 
-#ENTRYPOINT ["python3","trainer.py"]
+#ENTRYPOINT ["python3","trainer-main.py","--damage_name","crack","--max_iter","2400","--check_period","600","--thresh_test","0.3"]
+ENTRYPOINT ["python3","trainer-main-gpu.py","--damage_name","dent","--max_iter","1000","--check_period","500","--thresh_test","0.4","--num-gpus","4","--config-file","configs/COCO-InstanceSegmentation/mask_rcnn_R_101_FPN_3x.yaml","--batch_size","8"]
```
Comment (datnt88) [inline Dockerfile:54]: can we put the dent as parameter??

File: custom_test_net_MAP.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import detectron2
```
Comment (datnt88) [inline custom_test_net_MAP.py:1]: please give  more  docstring to explain what the script should do?

File: custom_test_net_DICE.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            pred = torch.sum(out['instances'].pred_masks,dim=0) > 0
+            pred = pred.cpu().detach().numpy()
+            pred=pred.astype(int)
+            #cv2.imwrite('im/pred'+str(i)+'.png',pred*255)
+            intersection = np.logical_and(mask, pred)
+            if len(np.unique(pred,return_counts=True)[1])>1:
+                ground=np.unique(mask,return_counts=True)[1][1]
+                pred_val=np.unique(pred,return_counts=True)[1][1]
+                dice_score = 2*np.sum(intersection) / (ground+pred_val)
+            #print(dice_score)
```
Comment (datnt88) [inline custom_test_net_DICE.py]: - remove #print
